.github/workflows/tests.yml

```diff
@@ -59,11 +59,8 @@ jobs:
           version: v1.56.2
           skip-cache: true
 
-      - name: install tern
-        run: go install github.com/jackc/tern@latest
-
       - name: Run unit tests
-        run: go test -v -race -covermode=atomic -coverprofile=coverage.txt -coverpkg=./... ./...
+        run: make unit-tests
```

Comment:
The makefile has a comment `re-implementing .github/workflows/tests.yml as close as possible` which probably needs updating now :)

Reply:
Thanks - good catch 🙈